packages/new-deploy/package.json

```diff
@@ -24,12 +24,14 @@
     "build": "tsc --build",
     "check": "tsc --noEmit",
     "prettier": "prettier --write ./src",
-    "lint": "eslint --fix ./src"
+    "lint": "eslint --fix ./src",
+    "deploy": "ts-node scripts/deploy.ts config/development/config.json config/development/overrides.json",
+    "check-deploy": "ts-node scripts/check.ts config/development/config.json"
```

Comment:
document how to invoke these and what their arguments are instead of making yarn script aliases

Reply:
https://github.com/nomad-xyz/monorepo/issues/140